src/main/java/org/jabref/gui/collab/EntryAddChangeViewModel.java

```diff
@@ -23,7 +25,7 @@ public EntryAddChangeViewModel(BibEntry diskEntry) {
     @Override
     public void makeChange(BibDatabaseContext database, NamedCompound undoEdit) {
         database.getDatabase().insertEntry(diskEntry);
-        undoEdit.addEdit(new UndoableInsertEntry(database.getDatabase(), diskEntry));
+        undoEdit.addEdit(new UndoableInsertEntries(database.getDatabase(), Collections.singletonList(diskEntry)));
```

Comment:
That makes sense. My original purpose was to discourage users from using the singular version, but it would make it more consistent in that I have a singular `insertEntry` method in `BibDatabase`.

Reply:
Would this just require a new constructor for `UndoableInsertEntries` or require a whole new class `UndoableInsertEntry`?

Reply:
A new constructor should be sufficient.